ray-operator/go.sum

```diff
@@ -650,6 +650,8 @@ k8s.io/client-go v0.19.2/go.mod h1:S5wPhCqyDNAlzM9CnEdgTGV4OqhsW3jGO1UM1epwfJA=
 k8s.io/client-go v0.19.14 h1:D3V4mC6lQN3ZaABhOznxZXSsxMR5cdChjeMP4/pY4H8=
 k8s.io/client-go v0.19.14/go.mod h1:iWwXK/etTTPyssCycvgdQ9d6i0k93HXLtksUQ6A67WE=
 k8s.io/code-generator v0.19.2/go.mod h1:moqLn7w0t9cMs4+5CQyxnfA/HV8MF6aAVENF+WZZhgk=
+k8s.io/code-generator v0.19.14 h1:EFU42Hj0+FBZciWedyRtFHsPu75IaWca0slGB1LlBqk=
```

Comment:
Shall we unify the `code-generator` version?

Reply:
@harryge00 

k8s.io/code-generator is pinned to v0.19.14. 0.19.2 is from some other indirect dependencies I think. That's impossible to unify all version in go.sum. Let's try to make sure we use same minor version in go.mod

![image](https://user-images.githubusercontent.com/4739316/140058928-cc58b5d2-5c63-4cab-856d-34fc86473018.png)